doc/installation/Android.md

```diff
@@ -38,11 +38,24 @@ En cas de soucis avec le JDK installer via `brew install --cask zulu11` et ajout
 </details>
 <br/>
 <details>
-  <summary>[M1 processor] Error while building the app on android : `Android Studio-- Cause: error=86, Bad CPU type in executable`</summary>
+  <summary>General error for building the app on Android Studio</summary>
 
-If you get this error on M1 Mac, installing Rosetta 2 should solve the issue. You can install it with this command : `softwareupdate --install-rosetta`.
+These are the general solutions for errors :
 
-Rosetta will allow applications requiring Intel processor to run on M1 Mac.
+on "project/android : `./gradlew clean`, clear the build directory
```

Comment:
Juste une petit typo ici il manque un apostrophe autour de project/android sinon tu peux remplacer par : run `cd android && ./gradlew clean` to clear the build directory. Je te laisse choisir :)